docs/index.md

```diff
@@ -538,14 +538,14 @@ In license-maven-plugin, each header style is defined by patterns to detect it a
 <additionalHeaders>
   <javadoc_style>
     <firstLine>/**</firstLine>
-    <beforeEachLine> * </beforeEachLine>
-    <endLine> */</endLine>
+    <beforeEachLine xml:space="preserve"> * </beforeEachLine>
+    <endLine xml:space="preserve"> */</endLine>
     <afterEachLine></afterEachLine>
     <!--skipLine></skipLine-->
     <firstLineDetectionPattern>(\s|\t)*/\*.*$</firstLineDetectionPattern>
     <lastLineDetectionPattern>.*\*/(\s|\t)*$</lastLineDetectionPattern>
     <allowBlankLines>false</allowBlankLines>
-    <isMultiline>true</isMultiline>
+    <multiline>true</multiline>
```

Comment:
Please remove any change regarding multiline: I've correctly fixed in https://github.com/mathieucarbou/license-maven-plugin/pull/247